fmdb.xcodeproj/project.pbxproj

```diff
@@ -104,13 +103,6 @@
 			remoteGlobalIDString = 82D4642419C7C8170006BDFE;
 			remoteInfo = SalesforceSDKCoreTestApp;
 		};
-		CE0021CA1C066C1F0079DCA4 /* PBXContainerItemProxy */ = {
```

Comment:
This and the random block changes below are due to other scheme changes in the SalesforceSDKCore target dependency.